### PR TITLE
Fixed RequestControllerSubsystemTestCase

### PR DESCRIPTION
**Pull request by honza-kasik**
* Fixed bugs in `toggleTrackIndividualEndpoints`
* Fixed attributes presence based on server mode, see https://github.com/wildfly/wildfly-core/blob/5ff6433a5f06051ce8c29c654e9957d463e9ce77/request-controller/src/main/java/org/wildfly/extension/requestcontroller/RequestControllerExtension.java and `private static Collection<AttributeDefinition> getAttributeDefinitions(boolean registerRuntimeOnly)` in https://github.com/wildfly/wildfly-core/blob/5ff6433a5f06051ce8c29c654e9957d463e9ce77/request-controller/src/main/java/org/wildfly/extension/requestcontroller/RequestControllerRootDefinition.java

### CODE REVIEW

**Comment by honza-kasik** [comment]
I reviewed the possibility to have `@Category` for class, but it doesn't work with `@Category(Shared.class)` for class and `@Category(Standalone.class)` for one test - it runs all of them in domain mode and if I would revert it it would be IMHO less clear which test runs when...

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/requestcontroller/RequestControllerSubsystemTestCase.java [hunk 1]
```diff
@@ -59,6 +59,7 @@ public static void afterClass() throws InterruptedException, TimeoutException, I
         }
     }
 
+    @Category(Standalone.class)
     @Test
     public void verifyActiveRequests() throws Exception {
```
Comment (rhatlapa) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/requestcontroller/RequestControllerSubsystemTestCase.java:64]: This test can be updated also for domain, the runtime attributes are specific for single server => you need to check it on the server level and not on profile level => all tests can remain shared for standalone and domain, just domain requires handling for single server in domain rather then whole profile.

**Review by rhatlapa — CHANGES_REQUESTED** [review]

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/requestcontroller/RequestControllerSubsystemTestCase.java (hunk 1 above)
Comment (honza-kasik) [reply basic/src/test/java/org/jboss/hal/testsuite/test/configuration/requestcontroller/RequestControllerSubsystemTestCase.java:64]: Maybe I am not looking everywhere but I do not see a way how to see active request on server in web console gui. That looks like a possible bug.

Comment (rhatlapa) [reply basic/src/test/java/org/jboss/hal/testsuite/test/configuration/requestcontroller/RequestControllerSubsystemTestCase.java:64]: Yes, I also believe it is a bug, I don't see it there either.

**Review by rhatlapa — APPROVED** [review]

**Comment by rhatlapa** [comment]
LGTM, merging. Note, there should be added test for the `active-requests` in domain even though the functionality is currently missing in HAL

**Pull request merged**